code/datums/uplink_items/uplink_general.dm

```diff
@@ -494,12 +494,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 /datum/uplink_item/stealthy_tools
 	category = "Stealth and Camouflage Items"
 
-/datum/uplink_item/stealthy_tools/chameleon_stamp
-	name = "Chameleon Stamp"
-	desc = "A stamp that can be activated to imitate an official Nanotrasen Stamp. The disguised stamp will work exactly like the real stamp and will allow you to forge false documents to gain access or equipment; \
-	it can also be used in a washing machine to forge clothing."
-	reference = "CHST"
-	item = /obj/item/stamp/chameleon
+/datum/uplink_item/stealthy_tools
+	category = "Stealth and Camouflage Items"
+
```

Comment:
```suggestion
```
Somehow you added this twice, this should fix the PR failure.

Reply:
Yeah i was trying to roll back to previous commit is this something i can do? Just to know in the future. And really thanks you i fell like you my parent, just helping me here with my firts PR.

Reply:
It would be simpler to just commit this suggested change, fortunately it was just 3 isolated lines

Reply:
and yes, you can roll back to previous commits, useful when something more messy happens